nodejs/week1/src/backend/app.js

```diff
@@ -0,0 +1,81 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const mealsArray = require("./data/meals");
+const reviewsArray = require("./data/reviews");
+const reservationArray = require("./data/reservations.json");
+
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+
+// meals
+const mealsReviewsArray = mealsArray.map((mealitem) => {
+  mealitem.reviews = reviewsArray.filter(
+    (element) => mealitem.id === element.mealId
+  );
+  return mealitem;
+});
+
+app.get("/meals", async (request, response) => {
+  response.send(mealsReviewsArray);
+});
```

Comment:
i think you forgot to modularize the routes: check line 183 in homework.md (#### Modularizing routes)

Reply:
I dont understand what is mean that.

Reply:
benjamin told me that he removed this task, so don't worry!